plugins/MauticEmailMarketingBundle/Form/Type/ConstantContactType.php

```diff
@@ -84,7 +88,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             $page    = 1;
         }
 
-        $builder->add('list', 'choice', [
+        $builder->add('list', ChoiceType::class, [
             'choices'  => $choices,
```

Comment:
Look at https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#heading=h.imucd6w2ou47

All the choice types must have the `'choices_as_values' => true,` option and the labels must be flipped with values.

Reply:
Good catch.
But I don't understand why I have to use `choices_as_values` option. It's deprecated in Symfony 3.
https://symfony.com/doc/3.0/reference/forms/types/choice.html#choices-as-values

Reply:
Because this way the labels and values will be switched so this PR should not get through manual testing.

Reply:
Thanks for the explanation. Fixed in d2c2f9bdb84b815fe0975078045550b16843412c